src/extension/src/Constants.py

```diff
@@ -84,6 +84,7 @@ class PatchOperationErrorCodes(EnumBackport):
         DEFAULT_ERROR = "ERROR"  # default error code
 
     ERROR_ADDED_TO_STATUS = "Error_added_to_status"
+    PYTHON_NOT_FOUND = "No python version found on the machine"
```

Comment:
After some thought about this -- "Python version could not be discovered for core invocation."  If we use the message as is, it appears as though the blame is being shifted to the machine environment configuration while that's not true since the code throwing the error is in Python itself.